rust/osm2lanes/src/lib.rs

```diff
@@ -21,3 +21,8 @@ pub mod transform;
 pub use transform::{lanes_to_tags, tags_to_lanes, LanesToTagsConfig, TagsToLanesConfig};
 
 mod test;
+
+#[cfg(target_arch = "wasm32")]
```

Comment:
I think another crate is overkill; it's not too much work to feature-gate a few dependencies and this modul

Reply:
maybe, I still think the separation helps in readability and separation of concerns?

I will do that work as part of https://github.com/a-b-street/osm2lanes/pull/60, if I still think it is needed.
It also has the added benefit of being a home for node js based integration tests